allure-results/000f4fb6-8725-4aac-a799-24717b23685c-attachment.json

```diff
@@ -0,0 +1,9 @@
+  "args": [
+    {
+      "element-6066-11e4-a52e-4f735466cecf": "08ffda24-86c3-469a-a181-f6ed5763264a",
+      "ELEMENT": "08ffda24-86c3-469a-a181-f6ed5763264a"
+    }
+  ]
+}
```

Comment:
Allure reports are bound conceptually to each test execution. They do not belong structurally to the repo (the repository should ideally contain only files that are necessary for running the project). Consider deleting them, then ignoring the whole `allure-results` directory